src/Library/Lod/LodReader.h

```diff
@@ -0,0 +1,27 @@
+#pragma once
+
+#include <memory>
+#include <string>
+#include <vector>
+
+#include "Library/Lod/LodVersion.h"
+#include "Library/Lod/Internal/LodDirectory.h"
+#include "Library/Lod/Internal/LodFile.h"
+#include "Utility/Memory/Blob.h"
+
+
+class LodReader final {
```

Comment:
i went with even more minimalistic Reader interface than discussed above
that will require some slight downstream change of how lloyds beacons logic work (hacked into original LODs logic) but that requires refactoring for a long time anyway

Reply:
by the way all LODs seem to have only single directory anyway, so I removed all directory mentions altogether and when accessing the files i literally take 0th directory all the time as the default

Reply:
added an overhead summary comment